seminar_7_test/task_36.py

```diff
@@ -0,0 +1,8 @@
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    for row in range(1, num_rows + 1):
+        for col in range(1, num_columns + 1):
+            print(operation(row, col), end="  ")
```

Comment:
Лучше использовать табуляцию.
```suggestion
            print(operation(row, col), end="\t")
```